### PR TITLE
Set content-type for correct encoding on pwsh

### PR DESCRIPTION
**Pull request by dimasahmad**
Fixes https://github.com/python-poetry/poetry/issues/4845#issuecomment-998228153

### CODE REVIEW

**Comment by dimasahmad** [comment]
@sdispater sorry to bother you, but can you look at this?

File: vercel.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 {
+  "headers": [
```
Comment (neersighted) [inline vercel.json:2]: Are the nested `headers` objects here really correct? Without even looking at the docs, this looks fishy.

Comment (dimasahmad) [reply vercel.json:2]: See https://github.com/dimasahmad/install.python-poetry.org/commit/7a01d87dc60b674e8e9f45c52841a2da92a8535c#commitcomment-62669502 for the resulting site with this configuration.

I had a hard time looking for how to implement the headers within vercel.json, but found the [vercel.json](https://github.com/vercel/vercel/blob/main/vercel.json) in vervel's repo contains the headers config.

File: vercel.json [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    {
+      "headers": [
+        {
+          "key": "Content-Type",
+          "value": "text/plain; charset=utf-8"
+        }
+      ],
+      "source": "/"
```
Comment (abn) [inline vercel.json:10]:
```suggestion
      "source": "/(.*\.py)?"
```

Think something like this should handle both cases.

Comment (dimasahmad) [reply vercel.json:10]: What cases?

Comment (abn) [reply vercel.json:10]: The file is served on both `/` and `/install-poetry.py`.

```console
$ curl -I https://install-python-poetry-org-git-fork-dimasah-147cc4-python-poetry.vercel.app/install-poetry.py
HTTP/2 200 
date: Tue, 08 Feb 2022 22:35:19 GMT
content-type: application/octet-stream
content-length: 26429
x-robots-tag: noindex
etag: W/"797f9af00b43d53af0d66b20351c89bb547a23337d2672fcfe2c46bdd512f491"
access-control-allow-origin: *
cache-control: public, max-age=0, must-revalidate
content-disposition: inline; filename="install-poetry.py"
accept-ranges: bytes
age: 4244447
x-vercel-cache: HIT
server: Vercel
x-vercel-id: fra1:fra1::2rf6t-1644359719413-b58430301767
strict-transport-security: max-age=63072000; includeSubDomains; preload
```

**Comment by dimasahmad** [comment]
@abn unfortunately they are considered invalid. Seems like they didn't support regex.

**Comment by abn** [comment]
That is weird since this seems to indicate it is valid.

https://github.com/vercel/vercel/blob/dc2d814d0f75aa19f2422ce747121b73507e9514/vercel.json#L28

File: vercel.json [hunk 3]
```diff
@@ -1,4 +1,15 @@
 {
+  "headers": [
+    {
+      "headers": [
+        {
+          "key": "Content-Type",
+          "value": "text/plain; charset=utf-8"
+        }
+      ],
+      "source": "/(.*\\.py)?"
```
Comment (abn) [inline vercel.json:10]: Looks like now we have the reverse problem.

```console
$ curl -I https://install-python-poetry-org-8wq4gtcf3-python-poetry.vercel.app/install-poetry.py
HTTP/2 200 
date: Wed, 09 Feb 2022 00:14:59 GMT
content-type: text/plain; charset=utf-8
...

$ curl -I https://install-python-poetry-org-8wq4gtcf3-python-poetry.vercel.app/
HTTP/2 200 
date: Wed, 09 Feb 2022 00:15:10 GMT
content-type: application/octet-stream
...
```

Comment (abn) [reply vercel.json:10]: Might just be easier to have two separate entries for `/` and `/*.py`.

Comment (dimasahmad) [reply vercel.json]: Right. I've tried variety of valid javascript regex syntax without any success.

**Comment by dimasahmad** [comment]
It should works now
```powershell
(Invoke-WebRequest -Uri  https://install-python-poetry-org-n52f2sev4-python-poetry.vercel.app/ -UseBasicParsing).Headers.'Content-Type'
```
```text
text/plain; charset=utf-8
```
```powershell
PS C:\> (Invoke-WebRequest -Uri  https://install-python-poetry-org-n52f2sev4-python-poetry.vercel.app/install-poetry.py -UseBasicParsing).Headers.'Content-Type'
```
```text
text/plain; charset=utf-8
```

File: vercel.json (hunk 3 above)
Comment (abn) [reply vercel.json]: I have a feeling vercel treats `/` as a special case. /shrug

**Review by abn — APPROVED** [review]
LGTM

**Pull request merged**